tests/datasets.py

```diff
@@ -114,8 +114,8 @@ class DATASET_OPTIONS_T(TypedDict):
         'openneuro': 'ds000248',
         'osf': '',
         'web': '',
-        'include': ['derivatives/freesurfer/subjects/sub-01'],
-        'exclude': ['derivatives/freesurfer'],
+        'include': ['derivatives/freesurfer'],
```

Comment:
We don’t need any freesurfer. Fsaverage should be fetched automatically by the pipeline

Reply:
Oh, so … what do we even need then? 😅 just the MEG?

Reply:
Changed it to downloading only the MEG for sub-01 now, and omitting the anat